localproxy.yml

```diff
@@ -0,0 +1,4 @@
+- name: CVMFS
+  hosts: cvmfslocalproxies
+  roles:
+    - cvmfs
```

Comment:
Should we add a short `README` file in the `roles/` directory to explain why the `cvmfs` submodule to `galaxyproject/ansible-cvmfs` is there?

Reply:
Good idea, done!